### PR TITLE
[REF] base_address_extended: assigned street format to country mx

### PR DESCRIPTION
**Pull request by suniagajose**
Summary
-------------

New format:
`%(street_name)s #%(street_number)s - %(street_number2)s`

Instead (format default):
`%(street_number)s/%(street_number2)s %(street_name)s`

TODO
--------

- [x] assigned street format to country mx

- [x] ~~base_address_extended: missed UserError import~~ (waiting for odoo fix)

- [x] ~~base_address_extended: avoid False when assign street format values~~ (waiting for odoo fix)

> to avoid explicit text 'False' when omitting values in address, like this example:
captura de pantalla de 2016-12-23 19-12-48

### CODE REVIEW

File: addons/base_address_extended/models/base_address_extended.py [hunk 1]
```diff
@@ -85,4 +86,5 @@ def _split_street(self):
             vals[field_name] = street_raw.rstrip(street_format[previous_pos:])
             #assign the values to the fields. Note that a write(vals) would cause a recursion since it would bypass the cache
             for k, v in vals.items():
-                setattr(partner, k, v)
+                if v is not False:
+                    setattr(partner, k, v)
```
Comment (nhomar) [inline addons/base_address_extended/models/base_address_extended.py]: what is the rationale of this change?

Comment (suniagajose) [inline addons/base_address_extended/models/base_address_extended.py]: is to avoid explicit text 'False' in address, like this example:
captura de pantalla de 2016-12-23 19-12-48

Although it should be:
``` python
setattr(partner, k, v or '')
```

**Comment by suniagajose** [comment]
hi @nhomar,

what about this to avoid the explicit 'False', that I mention in the description of this PR?
```python
for k, v in vals.items():
    value = v != 'False' and v or ''
    setattr(partner, k, value)
```

**Comment by nhomar** [comment]
I prefer not change the logic...

did odoo accepted your PR?

**Comment by suniagajose** [comment]
i left a comment in that line in PR: https://github.com/odoo/odoo/pull/13835
